nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -26,6 +26,13 @@
   before changing the package to `pkgs.stalwart-mail` in
   [`services.stalwart-mail.package`](#opt-services.stalwart-mail.package).
 
+- The `stalwart-mail` service now runs under the `stalwart-mail` user and group
+  instead of a `DynamicUser`, to avoid the automatic ownership change on its
+  large file store each time the service was started. This module change
+  requires administrators to manually move the state directory from
+  `/var/lib/private/stalwart-mail` to `/var/lib/stalwart-mail` and to
+  set the ownership of this directory and its content to `stalwart-mail`.
```

Comment:
```suggestion
- The `stalwart-mail` service now runs under the `stalwart-mail` system user
  instead of a dynamically created via `DynamicUser`, to avoid automatic ownership
  changes on its large file store each time the service was started.
  This change requires to manually move the state directory from
  `/var/lib/private/stalwart-mail` to `/var/lib/stalwart-mail` and to
  change the ownership of the directory and its content to `stalwart-mail`.
```